setup.py

```diff
@@ -37,7 +37,8 @@
         'backports.functools_lru_cache; python_version<"3.2"',
         'jedi>=0.14.1,<0.16',
         'python-jsonrpc-server>=0.1.0',
-        'pluggy'
+        'pluggy',
+        'ujson<=1.35'
```

Comment:
Why this condition?

Reply:
no real reason, just decided to softfreeze the version.